cmd/helm/create.go

```diff
@@ -106,6 +107,10 @@ func (c *createCmd) run() error {
 	if c.starter != "" {
 		// Create from the starter
 		lstarter := filepath.Join(c.home.Starters(), c.starter)
+		// If path is absolute, we dont want to prefix it with helm starters folder
+		if strings.HasPrefix(c.starter, "/") {
```

Comment:
Can you use `filepath.IsAbs` instead? https://golang.org/pkg/path/filepath/#IsAbs

Reply:
Fixed, also in v3 pr